portfoliograph/__init__.py

```diff
@@ -0,0 +1,84 @@
+from typing import Dict, Set, TYPE_CHECKING, NamedTuple
+from psycopg2.extras import DictCursor
+import networkx as nx
+
+if TYPE_CHECKING:
+    from dbtool import DbContext
+
+
+class NameNode(NamedTuple):
+    name: str
+
+
+class BizAddrNode(NamedTuple):
+    bizaddr: str
+
+
+class RegistrationInfo(NamedTuple):
+    reg_id: int
+    reg_contact_id: int
+
+
+def exportgraph(db: 'DbContext'):
+    g = nx.Graph()
+
+    with db.connection() as conn:
+        reg_bbl_map: Dict[int, Set[str]] = {}
+        cur = conn.cursor(cursor_factory=DictCursor)
+
+        print("Building registration -> BBL mapping.")
+        cur.execute("""
+            SELECT registrationid, bbl FROM hpd_registrations
+        """)
+        for reg_id, bbl in cur.fetchall():
+            if reg_id not in reg_bbl_map:
+                reg_bbl_map[reg_id] = set()
+            reg_bbl_map[reg_id].add(bbl)
+
+        print("Building graph.")
+        cur.execute("""
+            SELECT * FROM hpd_contacts
+            WHERE
+              type = any('{HeadOfficer,IndividualOwner,CorporateOwner}') AND
+              businesshousenumber != '' AND
+              businessstreetname != '' AND
+              firstname != '' AND
+              lastname != ''
```

Comment:
Just FYI @toolness, the SQL filtering we use for Worst Evictors (and that we used to generate test data at the last retreat) is a bit more inclusive than this logic. See below:
```sql
where 
  type = any('{HeadOfficer,IndividualOwner,CorporateOwner}')
  and (businesshousenumber is not null OR businessstreetname is not null)
  and length(concat(businesshousenumber,businessstreetname)) > 2
  and (firstname is not null OR lastname is not null)
```

Reply:
Thanks! I've added a TODO in the code for this for now

Reply:
Ok addressed this in 276015e!